build.cmd

```diff
@@ -197,6 +198,26 @@ exit /b 1
 
 REM Build CoreCLR
 :BuildCoreCLR
+:: Run Steps to Generate ETW specific infrastructure the 
+if not exist "%__GeneratedIntermediatesDir%" md "%__GeneratedIntermediatesDir%"
+if not exist "%__GeneratedIntermediatesDir%\inc" md "%__GeneratedIntermediatesDir%\inc"
+set "genetw=%__SourceDir%\scripts\genWinEtw.py"
+
+mc -h "%__GeneratedIntermediatesDir%\inc" -r "%__GeneratedIntermediatesDir%" -b -co -um -p FireEtw "%__SourceDir%\VM\ClrEtwAll.man"
+IF ERRORLEVEL 1 goto FailedToGenEtwMetadata 
+
+echo generating clretwallmain.h and etmdummy.h
+"%PythonPath%"  "%genetw%" --man "%__SourceDir%\VM\ClrEtwAll.man" --exc "%__SourceDir%\VM\ClrEtwAllMeta.lst" --eventheader "%__GeneratedIntermediatesDir%\inc\ClrEtwAll.h" --macroheader "%__GeneratedIntermediatesDir%\inc\clretwallmain.h" --dummy "%__GeneratedIntermediatesDir%\inc\etmdummy.h" 
```

Comment:
the logic for this check is present in https://github.com/ramarag/coreclr/blob/event_logging_automation/src/pal/tools/probe-win.ps1#L57